packages/wxt/src/core/utils/manifest.ts

```diff
@@ -458,7 +458,7 @@ function discoverIcons(
   return icons.length > 0 ? Object.fromEntries(icons) : undefined;
 }
 
-function addDevModeCsp(manifest: Browser.runtime.Manifest): void {
+function addDevModeCsp(manifest: Manifest.WebExtensionManifest): void {
```

Comment:
You can undo this change. The old code is correct in v0.20

```suggestion
function addDevModeCsp(manifest: Browser.runtime.Manifest): void {
```